lib/configloader.js

```diff
@@ -1,4 +1,6 @@
 const request = require('./request');
+const { ModuleMode } = require('./enums/ModuleMode');
+const { LoggerSeverity } = require('./enums/LoggerSeverity');
```

Comment:
why the enums located in two different files? fills unnecessary, maybe both under `constants`?

Reply:
There are gonna be tons of different enums most likely. I think each should be a separate file to be 100% clear where each one lives. If they're all under constants it'll be impossible to find things.